public/app/core/components/form_dropdown/FormDropdown.tsx

```diff
@@ -0,0 +1,192 @@
+import React from 'react';
+// import ReactDOM from 'react-dom';
+import Select from 'react-select';
+import { AsyncCreatable } from 'react-select';
+import _ from 'lodash';
+import $ from 'jquery';
+import 'react-select/scss/default.scss';
```

Comment:
move this import and the changes in _gf_form.scss to a new component file named form_dropdown.scss